src/server/context.ts

```diff
@@ -14,7 +14,11 @@ export const createContextInner = async ({
   req,
   res,
 }: trpcNext.CreateNextContextOptions) => {
-  const session = await unstable_getServerSession(req, res, nextAuthOptions);
+  const session = await unstable_getServerSession(
```

Comment:
Looks like we don't have to use unstable here anymore https://next-auth.js.org/configuration/nextjs#getserversession